### DISCUSSION TITLE
Self-hosted instance: storage use stuck at 0 MB

### DISCUSSION THREAD

**Discussion by flaki** [ente-io/ente#4268] [Q&A]
I have recently started playing with a self-hosted Ente instance, I given myself an extra storage add-on, and synced my phone's gallery (6000+ photos and videos, 50+ GB of data). Yet the storage use shows up as "0 MB" in both the desktop & mobile apps -- why? I would like to know the storage used by my synced gallery, even if technically I can give myself practically any amount of "quota" I like (as long as my server can spare the space).

![image](https://github.com/user-attachments/assets/d60aa0b5-db53-457e-94d9-0056f31b8668)

**Comment by mnvr** [comment]
There was a temporarily a bug on main for a few hours where the usage would get stuck, maybe you cloned the repo at that time. Try running the latest main to see if that helps.

**Reply by mnvr** [reply]
* Try running the museum from the latest main

**Reply by flaki** [reply]
Nope, just updated to the latest state of the repo and still seeing this. Any pointers how to debug this?

`usage` is coming back as 0 in the JSON response from `/users/details/v2`.
